data/brands/shop/optician.json

```diff
@@ -1173,16 +1173,87 @@
     {
       "displayName": "眼鏡市場",
       "id": "meganeichiba-3bd301",
-      "locationSet": {"include": ["jp"]},
+      "locationSet": {"include": ["jp", "tw"]},
```

Comment:
This is a Japanese brand, suggest those Chinese name in Japan may not a good idea. I think you can separate another entry for Taiwan.

Reply:
Sure, I've spilt it into 2 items, thanks. 👍